catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/data/impl/MetacardTypeImplTest.java

```diff
@@ -123,6 +129,27 @@ public void testEquals() {
     }
 
     @Test
+    public void testHashCode() {
+        MetacardTypeImpl metacardType1 = generateMetacardType("test", 0);
```

Comment:
But if I remember the metacardType comparison correctly the type string is a part of that, so perhaps the ones that rely on it should use a constant (or at least local variable).